src/gleam/dynamic.gleam

```diff
@@ -1462,3 +1462,13 @@ fn all_errors(result: Result(a, List(DecodeError))) -> List(DecodeError) {
     Error(errors) -> errors
   }
 }
+
+/// A decoder that always succeeds
+pub fn success(value) {
+  fn(_) { Ok(value) }
+}
+
+/// A decoder that always fails
+pub fn fail() {
+  fn(x) { Error([DecodeError("", found: classify(x), path: [])]) }
```

Comment:
Let's set the expected value here as the error is not valid without one.

Reply:
Just pass in as a string? I think in some usecases there isn't really an expected value. 
@hayleigh-dot-dev do you have opinions?

Reply:
There must always be a value.